app/jacoco_offline.gradle

```diff
@@ -0,0 +1,147 @@
+// https://stackoverflow.com/questions/53071373/powermock-jacoco-gradle-0-coverage-for-android-project?rq=1
+
+apply plugin: 'jacoco'
+
+configurations {
+    jacocoAnt
+    jacocoRuntime
+}
+
+jacoco {
+    toolVersion = "0.8.1"
+}
+
+def offline_instrumented_outputDir = "$buildDir.path/intermediates/classes-instrumented/debug"
+
+tasks.withType(Test) {
+    jacoco.includeNoLocationClasses = true
```

Comment:
Everything still works.  The run time is still the same too.
From additional research on this, I couldn't find any information on when exactly to turn it on vs when to turn it off.  However, I did read that Roboelectric requires that you have this enabled in order to get coverage with Roboelectric tests.  I do not use Roboelectric right now, but I think keeping this enabled is the right choice so if I ever add Roboelectric it would work straight away.

Reply:
I'd vote to remove it, as it's not needed and keeping something because *someday* you may use *some other* library is not a great reason.

Reply:
I have commented it out and left behind a comment about it.  Being that this is mostly a copy and pasted file, if I am making significant changes I would prefer to comment them out with an explanation.